samples/bootloader/README.rst

```diff
@@ -33,15 +33,17 @@ This is accomplished by the following steps:
      This metadata contains the full public key corresponding to the private key that was used to sign the firmware.
      The bootloader sample checks the public key against a set of provisioned keys.
      Note that to save space, only the hashes of the provisioned keys are stored, and only the hashes of the keys are compared.
-     If the public key in the metadata matches one of the provisioned public key hashes, the image is considered valid.
-     If the public key does not match any of the provisioned hashes, validation fails.
+     If the public key in the metadata matches one of the valid provisioned public key hashes, the image is considered valid.
+     All public key hashes at lower indices than the matching hash are permanently invalidated at this point, which means that images can no longer be validated with those public keys.
+     For example, if an image is successfully validated with the public key at index 2, the public keys 0 and 1 are invalidated.
+     If the public key does not match any of the still valid provisioned hashes, validation fails.
+     This mechanism can be used to decommission broken keys.
```

Comment:
What mechanism are you referring to? Making validation fail? Or the full concept of keys being invalidated?
If the latter, this sentence should be moved one line up.

Reply:
The full concept. Will move the sentence